bigquery/google/cloud/bigquery/job.py

```diff
@@ -931,13 +960,30 @@ def __init__(self):
     https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.copy.writeDisposition
     """
 
+    @property
+    def kms_key_name(self):
+        """See
```

Comment:
Add the type here and a one sentence description, too.

Reply:
This is consistent with the rest of the file